### ISSUE TITLE
Support/issue for uint's.

### ISSUE THREAD

**Issue by EricSmekens** [tjanczuk/edge#421]
I've changed the sample number 107 to include a uint:

``` javascript
// Overview of edge.js: http://tjanczuk.github.com/edge

var edge = require('../lib/edge');

var hello = edge.func(function () {/*
    async (input) => 
    {
        uint myuint = 3456;
        var result = new {
            anInteger = 1,
            aNumber = 3.1415,
            aString = "foobar",
            aBool = true,
            anObject = new { a = "b", c = 12 },
            anArray = new object[] { "a", 1, true },
            aBuffer = new byte[1024],
            aUint = myuint,
        };

        return result;
    }
*/});

hello(null, function (error, result) {
    if (error) throw error;
    console.log('-----> In node.js:');
    console.log(result);
});
```

This returns with (last part):

```
 aUint: '3456' }
```

Not sure if this an issue, but is there a reason why it's put there as a string?
Or is this an unknown issue?

**Comment by tjanczuk** [comment]
This seems related to https://github.com/tjanczuk/edge/issues/258

**Comment by EricSmekens** [comment]
This PR #299 does not solve this issue.

**Comment by EricSmekens** [comment]
@tjanczuk Seems not to be related, when I apply the PR for Node 5.9.1 and build it myself on Windows 10, there is no change in this scenario.

**Comment by EricSmekens** [comment]
Fixed by #428.
